src/App.vue

```diff
@@ -51,7 +51,7 @@
           </v-list-item>
           <v-list-item href="https://github.com/pathnirvana/tipitaka.lk" target="_blank">
             <v-list-item-icon><v-icon>mdi-github</v-icon></v-list-item-icon>
-            <v-list-item-title>කේත දත්ත ගබඩාව / GitHub</v-list-item-title>
+            <v-list-item-title>කේත කෝෂ්ඨය / ගිට්හබ්</v-list-item-title>
```

Comment:
Since most people are not familier with these words, I would suggest to keep this unchanged.

Reply:
සියල්ලටම සරල හෝ හුරුපුරුදු වචන සැපයීමට නොහැකියි. `repository` යන්න ද සංවර්ධන කටුයුතුවල නිරත වන පුද්ගලයින්ට මිස අන් අයට නුහුරු වචනයකි. එහි ගිට්හබ් යන්න සඳහන් බැවින් හඳුනා ගැනීම තවත් පහසු කරයි. වචන තිබියදී මෙවැනි වචන දෙකක් භාවිතා කිරීමට අනවශ්‍ය අතර මෙමගින් වෙනත් ප්‍රයෝජන සඳහා බහුලව යොදා ගැනෙන 'data store' යන්න අදහස් වේ. හෙළබස හරහා දැනටමත් පරිවර්තනය කර තිබෙන හා වෙමින් පවතින කේත කළමනාකරණ සේවා සහ මෘදුකාංග වලට මෙය ඇතුළත් කර ඇත.

![image](https://user-images.githubusercontent.com/80757159/224693732-15561011-b831-4a3c-be73-e19f166bcb84.png)

Reply:
yes you are correct this is only useful for developers or who are familiar with technology. So it is best to keep them in English. There is no point in keeping everything in Sinhala in my opinion.

Reply:
කණගාටුයි. සිංහල භාෂා සහාය සමස්ත ජනතාව සඳහා වන බැවින් මෙම යෝජනාව ද පිළිගැනීමට නොහැකියි. එයට අකැමැති නම් දායකත්‍වය සැපයීම අත්හිටුවීමට සිදුවේ. ඉහත පැහැදිලි කිරීම නැවත කියවා බලන්න. සහයෝගය දක්වන්න.✌️